.github/workflows/test-fakeredis.yml

```diff
@@ -0,0 +1,152 @@
+---
+name: Test Dragonfly/Fakeredis
+
+on:
```

Comment:
seems that you need 
`id-token:write`

directive like here: https://github.com/dragonflydb/dragonfly/blob/4835b5debc955a60e85d1137b6da0531ec8cc380/.github/workflows/docker-weekly.yml#L12

for html publish action to work

Reply:
I divided the workflow to several jobs (See [here](https://github.com/dragonflydb/dragonfly/actions/runs/10477664538)).

- Running tests (with matrix so each set of tests can be run separately, giving more visibility into what is working well vs. what isn't).
- Publishing HTML.
- Publishing workflow summary.

In the `Publishing HTML` part - I added `id-token:write` permission, so the permission applies only on the relevant part of the workflow.

Reply:
I will look into why it doesn't work here.

Reply:
Added the token, should be fine now.